free-tunes/Utilities/CustomTextField.swift

```diff
@@ -22,11 +22,17 @@ extension UITextField {
         )
     }
 
-    func applyDefaultStyle(withName: String) {
+    func applyDefaultStyle(withName: String?) {
         self.layer.borderColor = (UIColor.lightGray).cgColor
         self.layer.masksToBounds = true
         self.layer.borderWidth = 1.5
         self.layer.cornerRadius = 8
```

Comment:
Just a small enhancement. This function is doing too many things. I would rather place line 25,26,27 into a protocol with 3 functions - addBorderColor, addBorderWidth, addMaskedBounds. This protocol will then get implemented by your class, your class then calls these methods which goes into your extension where the actual functionality is. I would also do the same for the corenerRadius.

Please reach out if you need further explanation.

Reply:
i don't think I grasp fully the use of that implementation  as the function handles default styling of my textfields thus as it is  it's easily reusable